client/src/components/RichText/LinkControl.tsx

```diff
@@ -90,7 +90,11 @@ export const LinkControl = ({
   return (
     <div className={styles.linkControl}>
       <div onClick={signalExpandShowModal} className="rdw-option-wrapper">
-        <BiLink size="18px" />
+        <Tooltip label="Link" placement="bottom" openDelay={500} fontSize="xs">
+          <span>
+            <BiLink size="18px" />
+          </span>
+        </Tooltip>
```

Comment:
is this change meant to be in this PR? usually we try to scope PRs to a single set of changes, for a couple of reasons:
1. easier to review
2. if we need to revert commits, we can do so in isolation without affecting other changes